src/tendencies/implicit/wfact.jl

```diff
@@ -7,73 +7,52 @@ import ClimaCore.Spaces as Spaces
 import ClimaCore.Operators as Operators
 import ClimaCore.Fields as Fields
 
-# If ᶜρcₜ = -ᶜdivᵥ(ᶠwinterp(ᶜJ, ᶜρ) * ᶠu₃ * ᶠinterp(ᶜρc / ᶜρ)), then
-# ∂(ᶜρcₜ)/∂(ᶠw_data) =
-#     -ᶜdivᵥ_stencil(ᶠwinterp(ᶜJ, ᶜρ) * ᶠw_unit * ᶠinterp(ᶜc)) -
-#     ᶜdivᵥ_stencil(ᶠu₃) * ᶠinterp_stencil(1) * ∂(ᶜρc)/∂(ᶠw_data)
-# If ᶜρcₜ = -ᶜdivᵥ(ᶠwinterp(ᶜJ, ᶜρ) * ᶠupwind(ᶠu₃, ᶜρc / ᶜρ)), then
-# ∂(ᶜρcₜ)/∂(ᶠw_data) =
-#     -ᶜdivᵥ_stencil(ᶠwinterp(ᶜJ, ᶜρ) *
-#     ᶠupwind(ᶠu₃ + εw, ᶜρc / ᶜρ) / to_scalar(ᶠu₃ + εw)) -
-#     ᶜdivᵥ_stencil(ᶠinterp(ᶜρ)) * ᶠupwind_stencil(ᶠu₃, 1 / ᶜρ) *
-#     ∂(ᶜρc)/∂(ᶠw_data)
-# The εw is only necessary in case w = 0.
+# From the chain rule, we know that
```

Comment:
Is there any plan to move those comments into proper documentation? Or do people prefer to have them here?

Reply:
Yes, these will all turn into a proper documentation page during the implicit solver refactoring.